File: API/Helpers/UserParams.cs
namespace API.Helpers
{
    public class UserParams : PaginationParams
    {
        public string currentUsername { get; set; }
        public string Department { get; set; }
        public int MinAge { get; set; } = 0;
        public int MaxAge { get; set; } = 25;
        public string OrderBy { get; set; } = "lastActive";
    }
} 

File: client/src/app/_models/userParams.ts
import { User } from "./user";

export class UserParams {
    department: string;
    minAge = 0;
    maxAge = 25;
    pageNumber = 1;
    pageSize = 5;
    orderBy = 'lastActive';

    constructor(user: User) {
        this.department = user.department;
    }
}